tests/helpers.mjs

```diff
@@ -83,15 +95,15 @@ export const transfer = async ({
     },
     memory,
   );
-  assertNoResultError(transferResult);
```

Comment:
Did you intend to parameterize the error checking for this rather than simply removing here?

Reply:
OH i see true is the default for handle.